tests/functional/pages.py

```diff
@@ -408,6 +428,10 @@ def click_approved(self):
         element = self.wait_for_element(EditMeasureLocators.SEND_TO_APPROVED)
         self.scroll_and_click(element)
 
+    def click_update(self):
+        element = self.wait_for_element(EditMeasureLocators.UPDATE_MEASURE)
```

Comment:
Can we use `self.find_element_by_link_text("Update").click()` or `self.find_element(EditMeasureLocators.UPDATE_MEASURE).click()` here? I'm not sure why we need to wait for it?

Reply:
We could do that - although we don't really seem to do that in many places? Is that what we should be doing? It feels like something more specific is more valuable. If we decided the content for the link should be "Create new version", would we want the test to break? 🤔 I don't know.

I tried to be consistent with what we have. I'm happy for us to use words, but we should probably make a decision as I think you keep making comments like this on my tests and I'm not confident we ever came down hard on the right way to do it.